src/Entity/CitizenInitiative.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\EventRepository")
+ */
+class CitizenInitiative extends EventBase
+{
+    /**
+     * @ORM\Column(type="json_array")
+     */
+    private $interests = [];
+
+    /**
+     * @ORM\Column(type="boolean", options={"default": 0})
+     */
+    private $expertAssistanceNeeded;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(nullable=true, nullable=true)
+     *
+     * @Assert\Length(max=250)
+     */
+    private $expertAssistanceDescription;
+
+    /**
+     * @var Skill[]|Collection
+     *
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\Skill", inversedBy="citizenInitiatives", cascade={"persist"})
+     * @ORM\JoinTable(
+     *     name="citizen_initiative_skills",
+     *     joinColumns={
+     *         @ORM\JoinColumn(name="citizen_initiative_id", referencedColumnName="id", onDelete="CASCADE")
+     *     },
+     *     inverseJoinColumns={
+     *         @ORM\JoinColumn(name="skill_id", referencedColumnName="id", onDelete="CASCADE")
+     *     }
+     * )
+     *
+     * @Assert\Valid
+     * @Assert\Count(min=1, groups={"competences"})
+     */
+    private $skills;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default": 0})
+     */
+    private $coachingRequested;
+
+    /**
+     * @ORM\Embedded(class="CoachingRequest", columnPrefix="coaching_request_")
+     *
+     * @var CoachingRequest
+     */
+    private $coachingRequest;
+
+    public function __construct(
+        UuidInterface $uuid,
+        Adherent $organizer,
+        string $name,
+        EventCategory $category,
+        string $description,
+        PostAddress $address,
+        \DateTime $beginAt,
+        \DateTime $finishAt,
+        string $slug = null,
+        \DateTime $createdAt = null,
+        int $participantsCount = 0,
+        bool $coachingRequested = false,
+        CoachingRequest $coachingRequest
+    ) {
+        $this->uuid = $uuid;
+        $this->organizer = $organizer;
+        $this->setName($name);
+        $this->slug = $slug;
+        $this->category = $category;
+        $this->description = $description;
+        $this->postAddress = $address;
+        $this->participantsCount = $participantsCount;
+        $this->beginAt = $beginAt;
+        $this->finishAt = $finishAt;
+        $this->createdAt = $createdAt ?: new \DateTime('now');
+        $this->updatedAt = $createdAt ?: new \DateTime('now');
```

Comment:
Please use `\DateTimeImmutable` for those two, and maybe `$this->createdAt = $this->updatedAt = ...`